src/server/adminInviteUser.ts

```diff
@@ -88,11 +88,11 @@ export async function adminInviteUser(
     const { data: profileData, error: createProfileError } = await supabase
         .from("profiles")
         .insert({
-            primary_key: newUserData.user.id,
             role: userDetails.role,
             first_name: userDetails.firstName,
             last_name: userDetails.lastName,
             telephone_number: userDetails.telephoneNumber,
+            user_id: newUserData.user.id,
```

Comment:
Ooh, was this another user_id related bug?

Reply:
Yes - it wasn't making the connection to users table and not displaying the data in the users table

Reply:
oh cool - let's merge this asap so we can include it in today's release